lambdas/search/index.py

```diff
@@ -50,6 +73,26 @@ def lambda_handler(request):
         }
         size = 0
         _source = []
+    elif action == 'images':
+        body = {
+            'query': {
+                'terms': {
+                    "ext": IMG_EXTS
+                }
+            }
+        }
+        size = 10
```

Comment:
I think we should request 100 of these, e.g. https://open.quiltdata.com/b/nasanex. @nl0 can yo confirm they are lazy loaded?

Reply:
confirming